kalite/distributed/static/js/distributed/contentrec/hbtemplates/content-explore-topic.handlebars

```diff
@@ -3,17 +3,17 @@
   <div class="content-explore-topic">
     <div class="row">
       <div class="col-xs-12">
-          <h4><small>{{_ "Based on your interest in" }} {{ interest_topic.title }}:</small></h4>
+          <p class="up10">{{_ "Based on your interest in" }} <b>{{ interest_topic.title }}</b><span class="sr-only">{{_ ", we recommend you" }}</span>:</p>
       </div>
     </div>
     <div class="row">
       <div class="col-xs-5">
-          <h3>{{ suggested_topic.title }}:</h3>
+          <h2 class="h3">{{ suggested_topic.title }}:</h2>
       </div>
       {{#if suggested_topic.description }}
       <div class="col-xs-7">
-        <h4>{{_ "Description" }}:</h4>
-        <p>{{ truncate suggested_topic.description 70 }}</p>
+        <p class="h4">{{_ "Description" }}:</p>
+        <p>{{ truncate suggested_topic.description 100 }}</p>
```

Comment:
On some descriptions 70 chars wasn't even enough for a complete sentence, that's why I bumped it to 100 (to me at least sounded as too much of abrupt ending when listening the sr output). However, I thought to have made that change only with descriptions that get read as the `alt` attribute. Better solution would be to add `...` and `Read full description` link.